src/main/java/de/tinf/collections/ArrayUtils.java

```diff
@@ -7,6 +7,304 @@
  */
 public class ArrayUtils {
 
+    /**
+     * Converts an array of objects to an array of primitive values.
```

Comment:
_nitpick:_ What do you think about adding the types that are casted int the description? This would reflect in the docs. But I am also fine with a this kind of general description as these are quite many functions.